wpeverest/user-registration
# UR-1096 Feature - Limit length and minimum length feature.

### All Submissions:






### Changes proposed in this Pull Request:

Potential new feature to add Minimum Length and Limit Length settings in Input field and Textarea field.

Closes # .

### How to test the changes in this Pull Request:

1. User Registration > Add New > Create a Form > Drag Input / Textarea Field.
2. Click on respective field and under Advance Settings Toggle and Play with Limit Length and Minimum Length Options.
3. Check the validation words according to settings during Form Submission 

### Types of changes:




### Other information:




### Changelog entry

> Feature - Limit Length and Minimum Length advance settings.
